plugins/modules/remote_management/manageiq/manageiq_tags_info.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017, Daniel Korn <korndaniel1@gmail.com>
+# Copyright (c) 2017, Yaacov Zamir <yzamir@redhat.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+
+module: manageiq_tags_info
+version_added: 5.8.0
+short_description: Retrieve resource tags in ManageIQ.
+extends_documentation_fragment:
+- community.general.manageiq
+
+author: Alexei Znamensky (@russoz)
+description:
+  - The manageiq_tags_info module supports retrieving resource tags from ManageIQ.
+
+options:
+  resource_type:
+    type: str
+    description:
+      - The relevant resource type in manageiq.
+    required: true
+    choices: ['provider', 'host', 'vm', 'blueprint', 'category', 'cluster',
+        'data store', 'group', 'resource pool', 'service', 'service template',
+        'template', 'tenant', 'user']
+  resource_name:
+    type: str
+    description:
+      - The name of the resource at which tags will be controlled.
+      - Must be specified if I(resource_id) is not set. Both options are mutually exclusive.
+  resource_id:
+    description:
+      - The ID of the resource at which tags will be controlled.
+      - Must be specified if I(resource_name) is not set. Both options are mutually exclusive.
+    type: int
+'''
+
+EXAMPLES = '''
+- name: List current tags for a provider in ManageIQ
+  community.general.manageiq_tags_info:
+    resource_name: 'EngLab'
+    resource_type: 'provider'
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      username: 'admin'
+      password: 'smartvm'
+      validate_certs: false
+'''
+
+RETURN = '''
+'''
```

Comment:
There should at least be basic documentation on `tags`.

Reply:
done